packages/botonic-plugin-ai-agents/src/agent-builder.ts

```diff
@@ -26,17 +34,20 @@ export class AIAgentBuilder {
       instructions: this.instructions,
       tools: this.tools,
       outputType: OutputSchema,
+      inputGuardrails: this.inputGuardrails,
+      outputGuardrails: [],
     })
   }
 
   private addExtraInstructions(
     initialInstructions: string,
     contactInfo: ContactInfo
   ): string {
+    const instructions = `<instructions>\n${initialInstructions}\n</instructions>`
     const metadataInstructions = this.getMetadataInstructions()
     const contactInfoInstructions = this.getContactInfoInstructions(contactInfo)
     const outputInstructions = this.getOutputInstructions()
-    return `${initialInstructions}\n\n${metadataInstructions}\n\n${contactInfoInstructions}\n\n${outputInstructions}`
+    return `${instructions}\n\n${metadataInstructions}\n\n${contactInfoInstructions}\n\n${outputInstructions}`
```

Comment:
proposal (future PRs?): as instructions must be constructed a lot for defining behavior, maybe it would make sense to have a `InstructionsBuilder`?
something to be used like:
```
const instructions = new InstructionsBuilder()
  .add(new BaseInstructions("Step 1: Do something important"))
  .add(new MetadataInstructions())
  .add(new ContactInfoInstructions({ name: "John Doe", email: "john@example.com" }))
  .add(new OutputInstructions("Return result as JSON"))
  .build();
```